bundle/manifests/rhpam-kogito-operator.clusterserviceversion.yaml

```diff
@@ -268,6 +268,16 @@ spec:
                   valueFrom:
                     fieldRef:
                       fieldPath: metadata.annotations['olm.targetNamespaces']
+                - name: BUILDER_IMAGE
+                  value: rhpam-kogito-builder-rhel8
+                - name: RUNTIME_IMAGE
+                  value: rhpam-kogito-runtime-jvm-rhel8
+                - name: RUNTIME_NATIVE_IMAGE
+                  value: rhpam-kogito-runtime-native-rhel8
```

Comment:
we don't have native support on 1.5.x so I guess we should not document it in the manifests

```suggestion
```

Reply:
removed